Makefile

```diff
@@ -130,11 +131,22 @@ deploy-policy-framework-managed: kind-policy-framework-managed-setup deploy-poli
 .PHONY: deploy-community-policy-framework-managed
 deploy-community-policy-framework-managed: deploy-policy-framework-managed-crd-operator
 
+.PHONY: install-webhook 
+install-webhook:
+	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${CERT_MANAGER_VERSION}/cert-manager.yaml --kubeconfig=$(PWD)/kubeconfig_$(HUB_CLUSTER_NAME)
+	@echo "wait until cert-manager pods are up"
+	kubectl wait deployment -n cert-manager cert-manager --for condition=Available=True --timeout=180s --kubeconfig=$(PWD)/kubeconfig_$(HUB_CLUSTER_NAME)
+	kubectl wait --for=condition=Ready pod -l app.kubernetes.io/instance=cert-manager -n cert-manager --timeout=180s  --kubeconfig=$(PWD)/kubeconfig_$(HUB_CLUSTER_NAME)
+	curl https://raw.githubusercontent.com/stolostron/governance-policy-propagator/${RELEASE_BRANCH}/deploy/webhook.yaml | \
+		sed 's/namespace: open-cluster-management/namespace: $(KIND_HUB_NAMESPACE)/g' | \
+	 	kubectl apply -f- --kubeconfig=$(PWD)/kubeconfig_$(HUB_CLUSTER_NAME)
+	kubectl apply -f https://raw.githubusercontent.com/stolostron/governance-policy-propagator/$(RELEASE_BRANCH)/deploy/operator.yaml -n $(KIND_HUB_NAMESPACE) --kubeconfig=$(PWD)/kubeconfig_$(HUB_CLUSTER_NAME)
+
 .PHONY: kind-deploy-policy-framework
 kind-deploy-policy-framework:
 	@echo installing policy-propagator on hub
 	kubectl create ns $(KIND_HUB_NAMESPACE) --kubeconfig=$(PWD)/kubeconfig_$(HUB_CLUSTER_NAME)
-	kubectl apply -f https://raw.githubusercontent.com/stolostron/governance-policy-propagator/$(RELEASE_BRANCH)/deploy/operator.yaml -n $(KIND_HUB_NAMESPACE) --kubeconfig=$(PWD)/kubeconfig_$(HUB_CLUSTER_NAME)
+	$(MAKE) install-webhook
```

Comment:
Since the webhook and the propagator deployment are tied together, please rename the `install-webhook` target and (optionally) move the inital log and namespace creation over and make it a prerequisite for the target:
```
kind-deploy-policy-framework: kind-deploy-policy-propagator
```
At minimum, renaming it will make the purpose clearer.